components/wpt-runs.html

```diff
@@ -0,0 +1,146 @@
+<!--
+Copyright 2017 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+<link rel="import" href="../bower_components/polymer/polymer-element.html">
+<link rel="import" href="../bower_components/polymer/lib/elements/dom-if.html">
+<link rel="import" href="../bower_components/polymer/lib/elements/dom-repeat.html">
+
+<dom-module id="wpt-runs">
+  <template>
+    <style>
+      table {
+        width: 100%;
+      }
+      tr td {
+        padding: 0 0.5em;
+        margin: 2px;
+      }
+
+      .info {
+        padding: 0.5em;
+        background-color: #e2e3fd;
+        margin-bottom: 1em;
+        margin-top: 2em;
+        border-left: solid 4px #7477f4;
+      }
+      .missing {
+        background-color: #e2e3fd;
+      }
+      .present {
+        background-color: #e3fde2;
```

Comment:
These colors are a little confusing since green rectangles in the dashboard so far have only meant "tests passing." How about these:

```
.missing {
  background-color: #e3edfd;
}
.present {
  background-color: #337af3;
}
```

![screenshot from 2017-11-06 12 08 46](https://user-images.githubusercontent.com/916028/32461616-4c00161a-c2eb-11e7-92f7-51f08b101d92.png)